packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -65,6 +79,22 @@ export default function Sidebar({
     action: setOpenMobileSidebar,
   });
 
+  const trackSquadClicks = () => {
+    trackEvent({
+      event_name: 'click create squads',
```

Comment:
Fixed according to my changes in the taxonomy.
Please also add `target_id` as the variation of the button

```suggestion
      event_name: 'click create squad',
```